src/data/ons_pd_may_19.py

```diff
@@ -14,21 +14,23 @@ class ONSPostcodeDirectoryMay19(ONSPostcodeDirectory):
     :var dict ONSPostcodeDirectoryMay19.IMD_MAX: Highest ranked Lower Level Super Output Area (or equivalent) in each country
     :var dict ONSPostcodeDirectoryMay19.COUNTRY_CODES: ONS Postcode Directory codes for each country
     """
-    fields = ['lsoa11', 'msoa11', 'oslaua', 'osward', 'pcon', 'oscty', 'oseast1m', 'osnrth1m', 'lat', 'long', 'imd', 'ctry', 'rgn', 'pcd']
-    index_column = 'pcd'
+    # fields = ['lsoa11', 'msoa11', 'oslaua', 'osward', 'pcon', 'oscty', 'oseast1m', 'osnrth1m', 'lat', 'long', 'imd', 'ctry', 'rgn', 'pcd']
+    fields = ['lsoa11', 'msoa11', 'oslaua', 'osward', 'pcon', 'oscty', 'imd', 'ctry', 'rgn']
+    # index_column = 'pcd'
+    index_column = None
     data_types = {
         'oscty': 'category',
         'oslaua': 'category',
         'osward': 'category',
-        'oseast1m': 'Int32',
-        'osnrth1m': 'Int32',
+        # 'oseast1m': 'Int32',
```

Comment:
why are the coordinates commented out - is it because the default data type is correct?